.coveragerc

```diff
@@ -804,8 +804,8 @@ omit =
     homeassistant/components/mochad/switch.py
     homeassistant/components/modem_callerid/button.py
     homeassistant/components/modem_callerid/sensor.py
-    homeassistant/components/moehlenhoff_alpha2/__init__.py
     homeassistant/components/moehlenhoff_alpha2/climate.py
+    homeassistant/components/moehlenhoff_alpha2/coordinator.py
```

Comment:
Coverage on the coordinator is only on 37%, so we need to add it to coverage ignore.
```
---------- coverage: platform linux, python 3.12.3-final-0 -----------
Name                                                           Stmts   Miss  Cover   Missing
--------------------------------------------------------------------------------------------
homeassistant/components/moehlenhoff_alpha2/__init__.py           23      1    96%   44
homeassistant/components/moehlenhoff_alpha2/binary_sensor.py      24      0   100%
homeassistant/components/moehlenhoff_alpha2/button.py             20      1    95%   41
homeassistant/components/moehlenhoff_alpha2/climate.py            73     16    78%   83, 88, 94, 96, 106-109, 118-120, 124-130
homeassistant/components/moehlenhoff_alpha2/config_flow.py        31      0   100%
homeassistant/components/moehlenhoff_alpha2/const.py               4      0   100%
homeassistant/components/moehlenhoff_alpha2/coordinator.py        60     38    37%   50-51, 57-84, 91-128
homeassistant/components/moehlenhoff_alpha2/sensor.py             23      0   100%
--------------------------------------------------------------------------------------------
TOTAL                                                            258     56    78%
```